src/R/main.R

```diff
@@ -0,0 +1,123 @@
+##### Load libraries #####
+library(ggplot2)
+library(dplyr)
+library(DAAG) #datasets
+library(viridis) #colourmaps
```

Comment:
```suggestion
library(ggplot2)
library(dplyr)
library(DAAG)
library(viridis)
```

Reply:
Don't use comments unless you need to: document instead. (We can discuss what that means.)